### ISSUE TITLE
--mock has some kubeAPI calls that fail silently

### ISSUE THREAD

**Issue by VSharapov** [blackducksoftware/synopsys-operator#258]
**What happened**:
We had a kubeconfig with the wrong IP in the `server:` field
It took a long time (indicating there's an io timeout somewhere in there) but produced no errors.

**What you expected to happen**:
`--mock` command goes fast even with no connection to the serever

**How to reproduce it (as minimally and precisely as possible)**:
To reproduce, garble the `server:` in your kubeconfig and run a `--mock` command

**Screenshots/additional context**:

**Environment (please complete the following information):**
- Kubernetes version (use `kubectl version`): `Client Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.0+d4cacc0", GitCommit:"d4cacc0", GitTreeState:"clean", BuildDate:"2018-10-10T16:38:01Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"darwin/amd64"}`
- Openshift version (use `oc version`): N/A
- Cloud provider or hardware configuration: GCP set up through kippernetes
- OS (e.g: `cat /etc/os-release`): `cat: /etc/os-release: No such file or directory` hehheh, jk:
```
$ system_profiler SPSoftwareDataType
Software:

    System Software Overview:

      System Version: macOS 10.13.6 (17G5019)
      Kernel Version: Darwin 17.7.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Computer Name: vasiliys-mac
      User Name: Vasiliy Sharapov (vasiliys)
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 11 days 23:20
```
- Kernel (e.g. `uname -a`): `Darwin vasiliys-mac 17.7.0 Darwin Kernel Version 17.7.0: Thu Dec 20 21:47:19 PST 2018; root:xnu-4570.71.22~1/RELEASE_X86_64 x86_64`
- Others:

**Comment by VSharapov** [comment]
Still seeing it:
Valid kubeconfig server:
```
[0] vasiliys@vasiliys-mac:/tmp/bin$ time ./synopsysctl --insecure-skip-tls-verify --kubeconfig ~/Downloads/kubeconfig create blackduck bd --mock json >/dev/null                                                                                                                                                             
INFO[0000] creating Black Duck bd instance...

real    0m0.173s
user    0m0.032s
sys     0m0.018s
```
Invalid kubeconfig server
```
[0] vasiliys@vasiliys-mac:/tmp/bin$ time ./synopsysctl --insecure-skip-tls-verify --kubeconfig ~/Downloads/kubeconfig create blackduck bd --mock json >/dev/null                                                                                                                                                             
INFO[0030] creating Black Duck bd instance...

real    0m30.052s
user    0m0.035s
sys     0m0.021s
```
Using `synopsysctl` [build from yesterday (master)](https://github.com/blackducksoftware/synopsys-operator/commit/349005ec7a95f24d6dd5fd58eb2cc52c204f2441):
```
commit 349005ec7a95f24d6dd5fd58eb2cc52c204f2441                            
Refs: [master], {origin/master}, {origin/HEAD}                             
Author:     Michael Hammer <mike.p.hammer@gmail.com>                       
AuthorDate: Wed Jun 5 15:17:18 2019 -0400                                  
Commit:     operator-r2d2 <50117352+operator-r2d2@users.noreply.github.com>
CommitDate: Wed Jun 5 15:17:18 2019 -0400                                  
```

**Comment by mphammer** [comment]
Resolved by PR #425
